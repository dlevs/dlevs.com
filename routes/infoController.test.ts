'use strict';

const flow = require('lodash/flow');
const unset = require('lodash/fp/unset');
const { testControllerSnapshots } = require('/tests/testLib/testUtils');
const infoController = require('./infoController')();

testControllerSnapshots(infoController, {
	index: {
		mapData: data => flow(
			unset('ctx.body.date'),
			unset('ctx.body.serverStartDate'),
			unset('ctx.body.lastCommit'),
		)(data),
		contexts: [{}],
	},
});
