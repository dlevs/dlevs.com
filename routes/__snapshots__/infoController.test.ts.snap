// Jest Snapshot v1, https://goo.gl/fbAQLP

export const `index data passed to template with ctx {} 1`] = `
Object {
  "ctx": Object {
    "body": Object {
      "appVersion": "1.0.0",
      "environment": "test",
      "nodeVersion": "v12.3.1",
    },
    "query": Object {},
    "render": [Function],
    "state": Object {
      "CANONICAL_URL": "DUMMY_CANONICAL_URL",
    },
  },
  "renderedData": undefined,
}
`;
